README.md

```diff
@@ -364,7 +364,8 @@ var jobs = {
 
 **notes**
 
-- All plugins which return `(error, toRun)`.  if `toRun = false` on  `beforeEnqueue`, the job begin enqueued will be thrown away, and if `toRun = false` on `beforePerfporm`, the job will be reEnqued and not run at this time.  However, it doesn't really matter what `toRun` returns on the `after` hooks.
+- All plugins run a callback with arguments `(error, toRun)`.  if `toRun = false` on  `before_enqueue`, the job being enqueued will be thrown away, and if `toRun = false` on `before_perform`, the job will be reEnqueued and not run at this time. `toRun = false` does not affect execution in the `after_enqueue` hook.
+- When running multiple plugins on a job (with the array syntax), the plugins will be run sequentially (from let to right). If `toRun = false` in the `after_perform` hook, the next plugin will not be executed. Otherwise, `toRun` has no effect in the execution flow.
```

Comment:
The result data shouldn't effect the hooks at all.  Are you seeing something else?  If so, can you share an example?